src/index.ts

```diff
@@ -249,6 +249,14 @@ function ksmAddrDecoder(data: string): Buffer {
   return new Buffer(ss58Decode(data))
 }
 
+function ontAddrEncoder(data: Buffer): string {
+  return bs58Encode(Buffer.concat([Buffer.from([0x17]), data]))
+}
+
+function ontAddrDecoder(data: string): Buffer {
+  return bs58Decode(data).slice(1)
```

Comment:
If there's only one acceptable version, please check it and throw an error if appropriate.

Reply:
Check please